packages/internal/docs/docs/introduction/features/payments.mdx

```diff
@@ -3,22 +3,28 @@ title: Payments and subscriptions
 description: Integration with Stripe
 ---
 import useBaseUrl from "@docusaurus/useBaseUrl";
+import ProjectName from '../../shared/components/ProjectName.component';
+
+
+<ProjectName/> provides a payments and subscriptions functionality seamlessly integrated with Stripe.
+This feature empowers tenants to handle financial transactions and manage subscription plans directly within the application.
+This functionality is divided into two parts:
+
+- **[payments](#payments)**
+- **[subscription management](#subscription-management)**
 
-SaaS Boilerplate provides a payments and subscriptions functionality out of the box based on the integration with
-Stripe. This functionality is divided into two parts: **[payments](#payments)** and
-**[subscription management](#subscription-management)**.
 
 ### Payments
 
-The payments functionality allows users to define payment methods (including multiple cards) that are stored in the
-Stripe merchant account. Users can edit and update their payment methods at any time.
+The payments functionality enables tenants to define and manage payment methods securely stored in their Stripe merchant account.
```

Comment:
`functionality enables tenants to define and manage payment` sounds a bit weird to me. WDYT?